tirgei/Questioner
# #162978811 A user can login

#### What does this PR do?
This Pull Request creates a login page

#### Description of Task to be completed?
The tasks for this PR is:
- Create the login page 
- Style the login page

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner.git ```

CD into the project folder on your machine
```$ cd Questioner ```

Check out the branch for this PR
``` $ git checkout ft-user-login-162978811```

Open the `login.html` file on your browser

#### Pivotal Tracker story associated with this PR
[#162978811](https://www.pivotaltracker.com/story/show/162978811)

#### Screenshots
![login-page](https://user-images.githubusercontent.com/17829576/50698852-34782900-1057-11e9-8826-cbfe55a12ab8.png)


Mobile 
![login-mobile](https://user-images.githubusercontent.com/17829576/50698845-2f1ade80-1057-11e9-8f33-d428acd9f0fa.png)
